docs/source/development/add-ons/create-an-add-on-18.md

```diff
@@ -22,104 +25,104 @@ Follow the section {ref}`plone:create-project-cookieplone-system-requirements` t
 
 ## Generate the add-on project
 
-To develop an add-on for only the frontend, then run the following command to generate your add-on project using the `frontend_addon` Cookieplone template.
-To develop add-ons for each the frontend and backend that work together, then instead use the Cookieplone template `project` in the command.
```

Comment:
@silviubogan why did you remove this? It's quite up to date information and is valuable. Could you please bring it back?

Reply:
@sneridagh I moved it into the introduction, and rewrote it. We should tell the developer whether this page or another is the one they want to follow right away. @sneridagh please take another careful look, and let me know. @silviubogan please wait on this one to give @sneridagh a chance to review again, after he's had his morning ☕.